src/helm/benchmark/scenarios/melt_scenarios.py

```diff
@@ -499,3 +500,229 @@ def get_instances(self, output_path: str) -> List[Instance]:
                 instances.append(instance)
 
         return instances
+
+
+class MELTTextClassificationScenario(Scenario):
+    name = "melt_text_classification"
+    description = "Text Classification scenario."
+    tags = ["text_classification"]
+
+    def __init__(
+        self,
+        dataset_name: str,
+        revision: str,
+        subset: Optional[str] = "",
```

Comment:
Subset name should be `None` by default to follow the convention for `load_dataset()` ([doc](https://huggingface.co/docs/datasets/v3.6.0/en/package_reference/loading_methods#datasets.load_dataset)).